src/client/containers/Profile/index.js

```diff
@@ -23,7 +23,16 @@ export default function Profile() {
 
   return (
     <div className="profile-container">
-      {user && <ProfileDetails handleSubmit={handleSubmit} user={user} />}
+      {user && (
+        <ProfileDetails
+          handleSubmit={handleSubmit}
+          user={user}
+          profileImage={user.profileImage}
```

Comment:
If you already pass the entire user, why pass the other properties separately?

Reply:
I have tried using {...user}, but got an 'error: props spreading is not allowed'. So, I have taken them seperately.

Reply:
Didn't know that was not allowed. But still, why not use `user.profileImage` in the `ProfileDetails` component directly? Because you do have it there.